UnitsNet.Tests/CustomCode/ForcePerLengthTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \GenerateUnits.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
```

Comment:
@JKSnd You are right, the header doesn't make sense for these files. They are generated once (if the file does not already exist). I have updated the powershell script to change the contents of this header a bit,
but it is still a `<autogenerated>` type of header.

https://github.com/anjdreas/UnitsNet/commit/229bf9fe55d7a27aee0344e7440d95f6b473257b